util/asprintf.c

```diff
@@ -2,6 +2,8 @@
 #include <stdlib.h>
 #include <stdio.h>
 
+extern int vasprintf(char **ret, const char *format, va_list args)  __asm__("libutil_vasprintf");
```

Comment:
Looking at this closer, we could just make libutil_asprintf call libutil_vasprintf. Then no declaration is needed.